src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/SubscriptionAT.java

```diff
@@ -117,6 +110,59 @@ public void testOffsetsCommit() throws Exception {
         assertThat(committedOffset, equalTo("25"));
     }
 
+    @Test
+    public void testGetSubscriptionCursors() throws IOException {
+        final EventType eventType = createEventType();
+        final Subscription subscription = createSubscription(eventType);
+        commitCursors(subscription);
+
+        List<Cursor> actualCursors = getSubscriptionCursors(subscription);
+        Assert.assertEquals(Arrays.asList(new Cursor("0", "25")), actualCursors);
+    }
+
+    @Test
+    public void testGetSubscriptionCursorsEmpty() throws IOException {
+        final EventType eventType = createEventType();
+        final Subscription subscription = createSubscription(eventType);
+
+        Assert.assertTrue(getSubscriptionCursors(subscription).isEmpty());
+    }
+
+    @Test
+    public void testGetSubscriptionNotFound() throws IOException {
+        final EventType eventType = createEventType();
+        createSubscription(eventType);
+
+        given()
+                .get(format(CURSORS_URL, "UNKNOWN_SUB_ID"))
+                .then()
+                .statusCode(HttpStatus.SC_NOT_FOUND);
+    }
+
+    private Subscription createSubscription(EventType eventType) throws IOException {
+        final String subscriptionJson = "{\"owning_application\":\"app\",\"event_types\":[\"" + eventType.getName() + "\"]}";
+        final Response response = given()
+                .body(subscriptionJson)
+                .contentType(JSON)
+                .post(SUBSCRIPTIONS_URL);
+        return MAPPER.readValue(response.print(), Subscription.class);
+    }
+
+    private void commitCursors(Subscription subscription) {
+        // commit offsets and expect 200
+        given()
+                .body("[{\"partition\":\"0\",\"offset\":\"25\"}]")
+                .contentType(JSON)
+                .put(format(CURSORS_URL, subscription.getId()))
+                .then()
+                .statusCode(HttpStatus.SC_OK);
+    }
+
+    private List<Cursor> getSubscriptionCursors(Subscription subscription) throws IOException {
+        Response response = given().get(format(CURSORS_URL, subscription.getId()));
```

Comment:
final
Please update your branch with latest code from 'nakadi-jvm'
It has a fix for stylecheck task that will now check acceptance tests as well.